<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClientInfoPage</class>
 <widget class="QWizardPage" name="CClientInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Please Setup your P4 Client Information</string>
  </property>
  <property name="subTitle">
   <string>Client Name is automatically determined</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Build Directory:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>209</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="buildRelativeDirBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../MainWindow/MainWindow.qrc">
       <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Client Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Source Directory:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="sourceRelativeDir"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="clientDirBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../MainWindow/MainWindow.qrc">
       <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="clientDir"/>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="sourceRelativeDirBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../MainWindow/MainWindow.qrc">
       <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>P4 Client Directory:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="buildRelativeDir"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="clientName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="modelTechRelativeDir"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ModelTech Directory:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="modelTechRelativeDirBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../MainWindow/MainWindow.qrc">
       <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientDir</tabstop>
  <tabstop>clientDirBtn</tabstop>
  <tabstop>clientName</tabstop>
  <tabstop>sourceRelativeDir</tabstop>
  <tabstop>sourceRelativeDirBtn</tabstop>
  <tabstop>buildRelativeDir</tabstop>
  <tabstop>buildRelativeDirBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../MainWindow/MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
